packages/fxa-settings/src/components/ConfirmWithLink/index.tsx

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { useFtlMsgResolver } from '../../models/hooks';
+
+export type ConfirmWithLinkProps = {
+  confirmWithLinkPageStrings: ConfirmWithLinkPageStrings;
+  email: string;
+  // TODO : update type definition once callback function is defined
+  goBackCallback?: () => void;
+  isOpenWebmailButtonVisible?: boolean; // TO-DO: Replace broker functionality which gives us this value (provider?)
+  resendEmailCallback: () => void;
+};
+
+export type WebmailValues = {
+  buttonText: string;
+  link: string;
+};
+
+export type ConfirmWithLinkPageStrings = {
+  headingText: string;
+  headingFtlId: string;
+  instructionText: string;
+  instructionFtlId: string;
+};
+
+const ConfirmWithLink = ({
+  confirmWithLinkPageStrings,
+  email,
+  goBackCallback,
+  isOpenWebmailButtonVisible = false,
+  resendEmailCallback,
+}: ConfirmWithLinkProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const localizedAriaLabel = ftlMsgResolver.getMsg(
+    'confirm-signup-aria-label',
+    'An enveloppe containing a link'
```

Comment:
```suggestion
    'An envelope containing a link'
```

Also, not sure where this is in the .ftl world. I think this was the only instance I could find of `confirm-signup-aria-label`:

```sh
git grep -n "confirm-signup-aria-label" | cat

packages/fxa-settings/src/components/ConfirmWithLink/index.tsx:42:    'confirm-signup-aria-label',
```

Reply:
Good catch peter, I don't see that hooked up to an FTL file. 👀

Reply:
Interesting, I wonder if FtlMsgResolver's [`strict`](https://github.com/mozilla/fxa/blob/3045b6f2f44221117c2f264ba6494f95bbd43cc0/packages/fxa-react/lib/utils.tsx#L47-L74) mode would have caught this.